src/HealthChecks.AzureServiceBus/Configuration/AzureEventHubOptions.cs

```diff
@@ -0,0 +1,41 @@
+using Azure.Core;
+using Azure.Messaging.EventHubs;
+
+namespace HealthChecks.AzureServiceBus.Configuration;
+
+/// <summary>
+/// Azure Event Hub configuration options.
+/// </summary>
+public class AzureEventHubOptions
```

Comment:
```suggestion
public class AzureEventHubHealthCheckOptions
```

Options class name = Health check class name + Options  - just a convention to follow.